stage3/react/modules/module01/README.md

```diff
@@ -0,0 +1,38 @@
+# Неделя 1
+
+## React. Components
+
+!! Задание выполняется в личном приватном репозитории, НЕ в репозитории школы !!
+
+Что должно быть сделано:
+
+1) Создать отдельную ветку для этого задания
+2) Использовать CRA с темплейтом --typescript.
+3) Настроить eslint, prettier так, чтобы проект не собирался при наличие ошибок. 
+4) Добавить React-Router версии 6. Добавить хедер который показывает текущую страницу.
+5) Создать пустой React App и отрисовать следующее на главной странице:
+  - Search Bar. 
+  (примеры: https://studio.uxpincdn.com/studio/wp-content/uploads/2020/09/BlogHeader_SearchBar_1200x600.png, https://www.sliderrevolution.com/wp-content/uploads/2021/02/cssheader1.jpg)
+
+  Введенное значение сохранять в LocalStorage при анмаунте компонента. При инициализации его смотреть значение LocalStorage и отображать его.
+
+  - Cards. Чем больше деталей на карточке, тем лучше.  
+  (примеры: https://www.webdesignerdepot.com/cdn-origin/uploads/2017/01/behance.jpg, https://www.webdesignerdepot.com/cdn-origin/uploads/2017/01/rightmove.jpg, https://www.webdesignerdepot.com/cdn-origin/uploads/2017/01/awwwards.jpg).  
+6) Добавить тесты на компоненты
```

Comment:
6.1 Mock LocalStorage, test Search
6.2 Test single card
6.3 Test list of cards